modules/phase_field/examples/anisotropic_interfaces/snow.i

```diff
@@ -79,62 +67,70 @@
   [../]
 []
 
-[AuxKernels]
-  [./m]
-    type = ParsedAux
-    variable = m
-    args = T
-    constant_names = pi
-    constant_expressions = 4*atan(1)
-    function = '0.9*atan(10*(1-T))/pi'
-    execute_on = timestep_end
-  [../]
-[]
-
 [Materials]
   [./free_energy]
     type = DerivativeParsedMaterial
     block = 0
     f_name = fbulk
-    args = 'w m'
-    function = '1/4*w^4 - (1/2 - m/3)*w^3 + (1/4-m/2)*w^2'
-    outputs = out
+    args = 'w T'
+    constant_names = pi
+    constant_expressions = 4*atan(1)
+    function = 'm:=0.9 * atan(10 * (1 - T)) / pi; 1/4*w^4 - (1/2 - m/3) * w^3 + (1/4 - m/2) * w^2'
+    derivative_order = 2
+    outputs = exodus
   [../]
   [./material]
     type = InterfaceOrientationMaterial
     block = 0
-    c = w
+    op = w
   [../]
   [./consts]
     type = GenericConstantMaterial
     block = 0
-    prop_names  = 'M       '
+    prop_names  = 'M'
     prop_values = '3333.333'
   [../]
 []
 
+[Preconditioning]
+  [./SMP]
+    type = SMP
+    full = true
+  [../]
+[]
+
 [Executioner]
   type = Transient
-  solve_type = NEWTON
+  scheme = bdf2
+  solve_type = PJFNK
+  petsc_options_iname = '-pc_type -pc_hypre_type -ksp_gmres_restart -pc_hypre_boomeramg_strong_threshold'
+  petsc_options_value = 'hypre    boomeramg      31                 0.7'
```

Comment:
This is a poor threshold for a 2D problem. MOOSE provides good default boomeramg thresholds for both 2D and 3D problems. I recommend that you don't set this in any input file. You can confirm this by inspecting the header at the top of your simulation output. It'll tell you the threshold being used when using hypre.